src/components/meetups/MeetupDetailsPage.tsx

```diff
@@ -28,21 +28,23 @@ const styles = StyleSheet.create({
 
 export default function MeetupDetailsPage({ route }: MeetupDetailsPageProps) {
   // TODO: Load in real data of the passed in meeting
-  const { meetupID } = route.params;
-  const meetupTime = "3:00 PM";
-  const meetupDuration = "25";
-  const meetupLocation = "Online";
-  const meetupParticipants = ["John Smith", "Bob Smith", "Alice Smith"];
+  const { meetupID, timestamp, location, duration, userList } = route.params;
```

Comment:
We'll have to fix this part in the meeting. My code fetches the details upon navigating to the details page and looks like yours passed it in from the previous screen